cluster/points_writer.go

```diff
@@ -196,6 +202,13 @@ func (w *PointsWriter) WritePoints(database, retentionPolicy string, consistency
 	w.statMap.Add(statWriteReq, 1)
 	w.statMap.Add(statPointWriteReq, int64(len(points)))
 
+	if w.MaxConcurrentWrites > 0 {
```

Comment:
Makes sense, but this won't solve the issue that #6559 was intended to address though.  For example, a bad client sending thousands of huge `/write` payloads could still overwhelm the server by causing the server to use CPU to parse points as well as allocate memory for those points even if this limit is set in the `PointsWriter`.  Doing it in the handler allows us to drop the connection at the periphery and minimize damage.